packages/web/src/containers/swap-container/SwapContainer.tsx

```diff
@@ -1,6 +1,90 @@
-import React from "react";
-import Swap from "@components/swap/swap/Swap";
+import React, { useCallback, useState } from "react";
+import SwapCard from "@components/swap/swap-card/SwapCard";
+import { useAtom } from "jotai";
+import { CommonState } from "@states/index";
 
-const SwapContainer: React.FC = () => <Swap />;
+export interface SwapGasInfo {
+  priceImpact: string;
+  minReceived: string;
+  gasFee: string;
+  usdExchangeGasFee: string;
+}
+
+export const dummySwapGasInfo: SwapGasInfo = {
+  priceImpact: "-0.3%",
+  minReceived: "1.8445 ETH",
+  gasFee: "0.002451 GNOT",
+  usdExchangeGasFee: "$0.12",
+};
+
+export interface AutoRouterInfo {
```

Comment:
I think we need to refine the arguments for autorouters more. It would be nice to improve when working with data integration.

Reply:
This part bothered me. How data is patched Maybe now I put dummy data for the UI, but I think I need to improve it when working on data patching! thank you.